src/js/hooks/useWizard.js

```diff
@@ -0,0 +1,73 @@
+import { useMemo, useState } from 'react';
+
+import PropTypes from 'prop-types';
+
+const useWizard = ({ initialKey, steps }) => {
+  const [key, setKey] = useState(initialKey);
+
+  const set = (val) => {
+    setKey(val);
+  };
+
+  const first = () => {
+    setKey(steps[0]?.key);
+  };
+
+  const last = () => {
+    const lastIdx = steps.length - 1;
+    setKey(steps[lastIdx]?.key);
+  };
```

Comment:
It's cool that you added these but are we ever going to use them?
I don't care if the stay or they go, I am just not familiar with any use case where we will want to jump to first and last steps.

Reply:
We already have such use cases - e.g. if we have an SM in progress and it is supposed to be on "X" step, we go to step "X".

Reply:
yes, but I am not talking about function `set` but `last` and `first`

Reply:
maybe now we don't have the use case, but they felt cute and I also wanted to have a complete hook, that would cover any of the future cases, so I kept them :sadsmile: 
Let me know if I should remove them in your opinion guys.